CODEOWNERS

```diff
@@ -17,6 +17,7 @@ packages/metadata/* @raymondfeng
 packages/openapi-spec/*  @bajtos @jannyHou
 packages/openapi-spec-builder/* @bajtos @raymondfeng
 packages/openapi-v2/* @jannyHou
+packages/openapi-v3/* @jannyHou
```

Comment:
`openapi-v3-types`

Reply:
good catch, so weird...I remember clearly typed the correct name in this file, it's probably changed by a global replace :(